test/e2e_node/e2e_node_suite_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"flag"
+	"testing"
+)
+
+var host = flag.String("host", "localhost", "Address of the host being tested")
+var port = flag.Int("port", 4194, "Port of the application on the host being tested")
+
+func TestE2eNode(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2eNode Suite")
+}
+
+// Setup the kubelet on the node
```

Comment:
This does use ginkgo, there just isn't much setup teardown.  This is the suite generated by ginkgo bootstrap: http://onsi.github.io/ginkgo/#bootstrapping-a-suite.

The reason the node bootstrapping is in the test runner instead of the suite is so that the suite can run against a node that has already been bootstrapped.  We could run the runner in another ginkgo suite if you think it would be useful.